src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -1315,10 +1323,14 @@ describe("Shipping", () => {
     describe("with address verification enabled", () => {
       beforeAll(() => {
         ;(useFeatureFlag as jest.Mock).mockImplementation(
-          (featureName: string) => featureName === "address_verification"
+          (featureName: string) => featureName === "address_verification_us"
```

Comment:
The test update is not relevant to the change, and I'll need to follow up with a proper test.

This uses the correct feature flag name, and after correcting, tests are failing. It turns out the mock is leaking to other tests so I reset them.